runtime/jcl/common/shared.c

```diff
@@ -253,21 +253,23 @@ createCPEntries(JNIEnv* env, jint helperID, jint urlCount, J9ClassPathEntry*** c
 
 	for (i=0; i<urlCount; i++) {
 		UDATA cpeType = 0;
-		char* correctPath = NULL;
-
-		cpeType = getCpeTypeForProtocol((char*)urlArrayElements[i].protocolChars, urlArrayElements[i].protocolLen, urlArrayElements[i].pathChars, urlArrayElements[i].pathLen);
+		char *correctedPath = NULL;
+
+		cpeType = getCpeTypeForProtocol(
```

Comment:
While you're here, please add the missing spaces in the `for` loop.
Please reorder the locals here so `cpeType` can be initialized with its final value:
```c
	for (i = 0; i < urlCount; i++) {
		char *correctedPath = NULL;
		UDATA cpeType = getCpeTypeForProtocol(
```